### ISSUE TITLE
Voigt, Reuss, Hill averages for 2D systems

### ISSUE THREAD

**Issue by DBodesheim** [coudertlab/elate#6]
First of, thanks a lot for your work! I really like ELATE!

Now, that the two-dimensional properties are implemented, it might be useful to also include the Voigt, Reuss and Hill averaging schemes for 2D. I found this paper in which they derived them: https://doi.org/10.1016/j.eml.2019.100615  (eq. 6). (I also derived myself them and got the same result)

I'd be happy to implement this and make a pull request, if this is wanted.

**Comment by shahramyalameha** [comment]
It is surprising ...!!
The formalism employed in the two-dimensional systems seems to be directly lifted from the ElATools: https://github.com/shahramyalameha/ElATools code without proper attribution!  (Equations 27, 28, and 29 from ElATools [paper](https://www.sciencedirect.com/science/article/pii/S0010465521003076))

**Comment by fxcoudert** [comment]
@shahramyalameha the 2D code was implemented by reviewing equations available in all available literature, and after that it was indeed compared against existing implementations, including ElATools. I have now added a link to ElATools, and the associated paper.

@DBodesheim I think the 2D "average" moduli (including the area modulus) are less commonly used that for 3D. But I'll look into adding them at the next opportunity, thanks for the suggestion.
